[DOCS-8024 to 8028] Add new Obs Pipelines component docs (#23623)
DataDog/documentation: layouts/shortcodes/observability_pipelines/configure_log_archive/google_cloud_storage/instructions.md

### Create a storage bucket

1. Navigate to [Google Cloud Storage][9091].
1. On the Buckets page, click **Create** to create a bucket for your archives..
1. Enter a name for the bucket and choose where to store your data.
1. Select **Fine-grained** in the **Choose how to control access to objects** section.

> 1. Do not add a retention policy because the last data needs to be rewritten in some rare cases (typically a timeout case).

Reviewer: ```suggestion
1. Do not add a retention policy because the most recent data needs to be rewritten in some rare cases (typically a timeout case).
```

1. Click **Create**.

### Create an API key

If you don't already have an API key, create one to give the Observability Pipelines Worker access to your storage bucket.

1. Navigate to Google Cloud Credentials.

> 1. Click **Create credentials**. This is the API key that you use when you install the Observability Pipelines Worker later.

Reviewer: ```suggestion
1. Click **Create credentials**. You will use the generated API key when you install the Observability Pipelines Worker.
```
